Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -1009,10 +1009,5 @@ class FooAT11 < Formula
         expect(fa.problems).to be_empty
       end
     end
-
-    # include_examples "formulae exist", described_class::VERSIONED_KEG_ONLY_ALLOWLIST
-    include_examples "formulae exist", described_class::PROVIDED_BY_MACOS_DEPENDS_ON_ALLOWLIST
```

Comment:
We're checking these formulae exist a different way now, right? I'm 99% sure we are but just checking 😀

Reply:
Yep! `brew audit` now checks these on a per-tap basis rather than `brew tests`. It's part of [`tap_auditor.rb`](https://github.com/Homebrew/brew/blob/d7f03bc2dd61e3495f0b48b6c8371f567c215304/Library/Homebrew/tap_auditor.rb#L52-L75).